ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -31,55 +31,24 @@ Flow {
 
     spacing: 12
 
-    //TODO: remove once GIF previews are unfurled sender side
-       Repeater {
-        id: tempRepeater
-        visible: !RootStore.neverAskAboutUnfurlingAgain
-        model: linksModel
+    QtObject {
+        id: d
+        property bool localAskAboutUnfurling: true
+    }
 
-        delegate: Loader {
-            id: tempLoader
+    //TODO: remove once GIF previews are unfurled sender side
 
-            required property var result
-            required property string link
-            required property int index
-            required property bool unfurl
-            required property bool success
-            required property bool isStatusDeepLink
-            readonly property bool isImage: result.contentType && result.contentType.startsWith("image/") ? true : false
-            readonly property bool isUserProfileLink: link.toLowerCase().startsWith(Constants.userLinkPrefix.toLowerCase())
-            readonly property string thumbnailUrl: result && result.thumbnailUrl ? result.thumbnailUrl : ""
-            readonly property string title: result && result.title ? result.title : ""
-            readonly property string hostname: result && result.site ? result.site : ""
-            readonly property bool animated: isImage && result.contentType === "image/gif" // TODO support more types of animated images?
+    Loader {
+        visible: root.gifLinks && root.gifLinks.length > 0
+                 && !RootStore.gifUnfurlingEnabled
+                 && d.localAskAboutUnfurling && !RootStore.neverAskAboutUnfurlingAgain
+        sourceComponent: enableLinkComponent
```

Comment:
The `enableLinkComponent` is not working. The enable link preview component is not visible when GIFs are disabled from Settings, instead probably the Loader keeps the previous size and the message row height is too large.

<img width="1327" alt="Screenshot 2023-10-17 at 13 26 11" src="https://github.com/status-im/status-desktop/assets/47811206/8f0c96a6-6605-4659-9876-bac6b707c9de">

Reply:
oggghh, this was a tough one.

https://github.com/status-im/status-desktop/assets/25482501/36e8fba5-b75b-4cd3-b4b6-a7a733eaa908

The missing line turned out to be this one:
https://github.com/status-im/status-desktop/blob/452079d4e0e0578f31dce32c3dc0c3520dafb0e7/ui/StatusQ/src/StatusQ/Components/StatusMessage.qml#L360

I also changed the logic inside `LinksMessageView`, but that wasn't enough.

For some reason, when we take an item that is inside a `Loader`, and change it's `height` to `0`, the `Loader`'s height is automatically kept to be the initial `Loader.height` value.
`0` is a special value here. Changing it to ANY other value, e.g. even `1`  works, but setting item's `height` to `0` doesn't.

____
NOTE: For testing I hardcoded link previews into storybook's `StatusMessagePage` (you can see that in thevideo). But I don't think it makes sense to commit it, it's will take too much time to make it clean, I don't think it's worth it.